pkgs/applications/editors/vscode/extensions/default.nix

```diff
@@ -47,6 +47,21 @@ let
         };
       };
 
+      _2gua.rainbow-brackets = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          publisher = "2gua";
+          name = "Rainbow Brackets";
+          version = "0.0.6";
+          sha256 = "3svGVbaxUh/60NywMdWFsL3fSv+kPgt39bGmPVIu6sI=";
+        };
+        meta = with lib; {
+          description = "Provide rainbow colors for the round brackets, the square brackets and the squiggly brackets. This is particularly useful for Lisp or Clojure programmers, and of course, JavaScript, and other programmer";
```

Comment:
Description is too long.
https://nixos.org/manual/nixpkgs/unstable/#var-meta-description